### PR TITLE
Fixes #336 - Added rails commands for setting a channel to trusted

### LINKED ISSUE #336: Add console commands for trusted channels
As this in general can be a usefull thing, we should also document it.

See commit
https://github.com/zammad/zammad/commit/1e8616151c308fd9ad0dbd72b1c6ee6385054a9b#diff-836a9832d3b8bf7760d717ff5b434180R29

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

**Comment by ralf401** [comment]
@mgruner can you have a look? Thanks!!

File: channels/email/email-headers.rst [hunk 1]
```diff
@@ -158,3 +159,46 @@ email articles.
...
+
+.. code-block:: ruby
+
+   >> options = channel[:options]
+
+Add the ``"trusted"=>true`` flag for the inbound part of the channel:
+
+.. code-block:: ruby
+
+   >> options[:inbound][:trusted] = true
```
Comment (mgruner) [inline channels/email/email-headers.rst:201]: This looks good, but I believe that the settings will not be saved. Try it by reopening the Rails console and checking again.
`channel.save!` should do the trick.

**Review by mgruner — APPROVED** [review]
Looks good, but saving is still missing and needs to be corrected.

File: channels/email/email-headers.rst (hunk 1 above)
Comment (ralf401) [reply channels/email/email-headers.rst:201]: Thank you! Worked :-)

**Pull request merged**